### ISSUE TITLE
version is not set to AUTO when omitted.

### ISSUE THREAD

**Issue by umrashrf** [scrapinghub/doc.scrapinghub.com#144]
There http://doc.scrapinghub.com/shub.html#configuration

scrapinghub.yml requires explicit version.

```
Packing version None
Deploying to Scrapy Cloud project "......"
Error: Deploy failed (400):
missing 'version' argument
```

**Comment by redapple** [comment]
@umrashrf , @jdemaeyer , is it a bug on shub or a documentation issue?

**Comment by jdemaeyer** [comment]
hmm I'm not sure. The documentation is definitely "as it should be" and from what I can reproduce that is also how `shub` behaves (I have no problems deploying when there's no version set in `.scrapinghub.yml`). Could you paste the problematic `.scrapinghub.yml` @umrashrf?

**Comment by umrashrf** [comment]
I recently upgraded Ubuntu which broke all my python environments so I had to recreate my environment where I had this problem.

I just created a new environment in that same directory having scrapy.cfg and installed shub (2.2.0 and 2.3.0) and it seems to work as expected without the explicit version config with both versions of shub. AFAIR it was shub 2.2.0 but not completely sure.

This is really strange. Closing the issue.
